VOID/fitness/threshold.py

```diff
@@ -1,16 +1,25 @@
 import numpy as np
-
+import argparse
 from .base import Fitness
-
+import ipdb
 
 THRESHOLD = 1.5
+THRESHOLD_CATAN = 2.0
```

Comment:
Is this threshold in &#8491;? Keeping cation-anion distances below 2 &#8491; seems very restrictive to me, especially since the other threshold is 1.5, but maybe I am misinterpreting these threshold values.

Reply:
You're absouletly right, I set this as a default for trials, then moved to parse htvs arguments and forgot about it. Changed it to 3.5 as a default value